tools/liveimage-mount

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/python
-# coding: utf-8
+# coding: utf_8
```

Comment:
This is not a valid reference, per [PEP 263](https://www.python.org/dev/peps/pep-0263/).